src/reducers/profile-view.js

```diff
@@ -448,8 +443,15 @@ export const getDisplayRange = createSelector(
 /**
  * Profile
  */
-export const getProfile = (state: State): Profile =>
+export const getProfileOrNull = (state: State): Profile | null =>
```

Comment:
Why this function ? Do you plan to use it in your next PR?

Reply:
This has more to do with component work where there were logical issues with assuming that there was a valid profile. I changed the profile reducer type signature to be `Profile | null`. `getProfileOrNull` returns the actual data, while `getProfile` throws an error if there is no profile, as it assumes there was one there. I updated it to use the ensureExists function.